### ISSUE TITLE
ReduxDevTools throwing exception when initial state contains Double.NaN

### ISSUE THREAD

**Issue by joris628** [mrpmorris/Fluxor#503]
I have a State object, that contains a property of type `System.Double` witch is initialized with `Double.NaN`.
When I run the application using the ReduxDevTools enabled, I get the following exception:
```
 Unhandled exception rendering component: Store initialization error
      Fluxor.Exceptions.StoreInitializationException: Store initialization error
       ---> System.ArgumentException: .NET number values such as positive and negative infinity cannot be written as valid JSON. To make it work when using 'JsonSerializer', consider specifying 'JsonNumberHandling.AllowNamedFloatingPointLiterals' (see https://docs.microsoft.com/dotnet/api/system.text.json.serialization.jsonnumberhandling).
         at System.Text.Json.ThrowHelper.ThrowArgumentException_ValueNotSupported()
         at System.Text.Json.Utf8JsonWriter.WriteNumberValue(Double value)
         at System.Text.Json.Serialization.JsonConverter`1.TryWrite(Utf8JsonWriter writer, T& value, JsonSerializerOptions options, WriteStack& state)
```

After searching for a while, it seems that you used to be able to configure the JSON Serializer. However, by removing the support for `NewtonSoft.Json` it seems that you also no longer can configure the JSON Serializer for `System.Text.Json`.
[Removed dependency newtonsoft.json](https://github.com/mrpmorris/Fluxor/commit/5cf430e8a4512ba5307ea12dda11f35874fcf838#diff-2fc8be43f455b49b8847d0227e48900dc4ac742d3df54d646d4f6ee6ff3c7ebd)

Like the exception mentions, I should be able to set `JsonNumberHandling.AllowNamedFloatingPointLiterals`. Is there another way to achieve this?

**Comment by rodrigochapeta** [comment]
Same, before I just ignored errors for Newtonsoft, it was great.

**Comment by mrpmorris** [comment]
Fixed in 6.1
